src/game.cpp

```diff
@@ -9726,7 +9726,7 @@ int game::list_monsters(const int iLastState)
                         int numw = iMonsterNum > 9 ? 2 : 1;
                         mvwprintz(w_monsters, y, width - (6 + numw),
                                   (selected ? c_ltgreen : c_ltgray), "%*d %s",
-                                  numw, trig_dist(0, 0, critter->posx() - u.posx(),
+                                  numw, square_dist(0, 0, critter->posx() - u.posx(),
```

Comment:
You probably want `rl_dist`, which behaves like `square_dist` or `trig_dist`, depending on the "Circular distances" option.